pywbemtools/pywbemcli/_display_cimobjects.py

```diff
@@ -478,10 +478,45 @@ def _display_instances_as_table(insts, table_width, table_format,
     else:
         max_cell_width = table_width
 
-    header_line = []
+    # TODO: Decide whether and how the showing of units should be controlled.
+    show_units = True
+
+    if show_units:
```

Comment:
Obviously we need to account for servers that do not have class repos but I assume that is for after we commit this

Reply:
DONE - added code to detect that and in that case, the showing of units is silently given up.